tools/src/launcher/Main.hs

```diff
@@ -515,11 +523,73 @@ spawnNode (path, args, mbLogPath) = do
             logInfo "Node has started"
             return (ph, asc)
 
-runWallet :: (FilePath, [Text]) -> M ExitCode
-runWallet (path, args) = do
-    logNotice "Starting the wallet"
-    view _1 <$>
-        liftIO (readProcessWithExitCode path (map toString args) mempty)
+runWallet :: Bool -> (FilePath, [Text], Maybe FilePath) -> M ExitCode
+runWallet shouldLog (path, args, mLogPath) = do
+    customLogger LInfo NWallet "Starting the wallet"
```

Comment:
Why do you print it using `customLogger` now? This message corresponds to the launcher's logic, so it should still be printed using `logNotice` (i. e. launcher's logging).

Reply:
Done